cli/command/task/print.go

```diff
@@ -93,18 +114,21 @@ func Print(dockerCli *command.DockerCli, ctx context.Context, tasks []swarm.Task
 			taskErr = fmt.Sprintf("\"%s\"", taskErr)
 		}
 
-		fmt.Fprintf(
-			writer,
-			psTaskItemFmt,
-			indentedName,
-			task.Spec.ContainerSpec.Image,
-			nodeValue,
-			command.PrettyPrint(task.DesiredState),
-			command.PrettyPrint(task.Status.State),
-			strings.ToLower(units.HumanDuration(time.Since(task.Status.Timestamp))),
-			taskErr,
-		)
+		if quiet {
+			fmt.Fprintln(out, indentedName)
```

Comment:
>In addition, if quiet, maybe we do not have to process the err things. Just print when constructing of indentedName finishes. 😄

🤔 Yeah, there is no need to process errs.

>I think `indentedName = fmt.Sprintf(" \\_ %s", indentedName)` above may be displayed via this line of code. If that, I have no idea if it is reasonable. WDYT @gaocegege

Ah, okay I think we should pass the indent logic when `ps -q`

```
		if quiet {
			fmt.Fprintln(out, indentedName)
			continue
		}
		// Indent the name if necessary
		indentedName := name
		logrus.Debugf("DEBUG: %d, %d", prevSlot, task.Slot)
		// Since the new format of the task name is <ServiceName>.<Slot>.<taskID>, we should only compare
		// <ServiceName> and <Slot> here.
		if prevServiceName == serviceName {
			indentedName = fmt.Sprintf(" \\_ %s", indentedName)
		}
		prevServiceName = serviceName
		prevSlot = task.Slot
``` 

Then the result would be 

```diff
redis.1.mwvj0azq00lcydntsv79uj2k6
-  \_ redis.1.zk9fhepa7cgnoex3pgn0sngir
+ redis.1.zk9fhepa7cgnoex3pgn0sngir
```
```

Or do you have a better idea about indented name?